addons/stock/views/product_strategy_views.xml

```diff
@@ -22,14 +22,17 @@
                        force_save="1"/>
                 <field name="package_type_ids" string="Package type"
                        options="{'no_create': True, 'no_open': True}"
-                       groups="stock.group_tracking_lot" widget="many2many_tags"/>
-                <field name="location_out_id" string="Store to"
+                       groups="stock.group_tracking_lot" widget="many2many_tags"
+                       optional="show"/>
+                <field name="location_out_id" string="Store to sublocation"
```

Comment:
Why rename some label in the view and not directly in the model ? (I only find the tree view).